capabilities/communication/email-templates/maps/postmark.suma

```diff
@@ -25,7 +25,7 @@ map ListTemplates {
 }
 
 map GetTemplateContent {
-  http GET "/templates/{input.id}" {
+  http GET "/templates/{input.id.toString()}" {
```

Comment:
this is not correct, path parameters are not jessie, but simple variable access

Reply:
But it works. :D I needed to make linter happy as id is without type and it expects string. 

What is right way?

Reply:
If you specify string type to `field id`, linting will pass. It's weird tho, linter returns warning when used without `toString()`, it should not throw, right? @Jakub-Vacek

Reply:
But I don't want to have specified type on field. Thats whole point.

Reply:
Then I would expect that cli lint would not throw on warnings. 🤷🏻‍♂️

Reply:
So only way how to shut linter here is to specify type to id?

<img width="831" alt="image" src="https://user-images.githubusercontent.com/959390/140043459-e862b0e0-c6ce-4842-a9e5-24e9ac830753.png">

Reply:
Even if cli doesn't throw I should be able to do something with the warning.

Reply:
I guess you should specify type that can go in URL, it's linter's role to warn you that you're putting invalid variable into url, but in current state, linter doesn't have constant folding implemented, so you can specify variable `inputID = input.id` before http request and use it in URL without warnings.

Reply:
if we allow to have input without type, we need to allow type assertion in maps, to be able to handle it. This way it is not really easy to use.

Reply:
I think I fixed it. Can you check it?

Reply:
@martinalbert ScalarStructure should pass when string is expected

Reply:
also i think linter should just check the format with regex here, what's not string is cast to string anyway

Reply:
@lukas-valenta it passes, but it returns warning. Does it mean that when object is passed it will stringify that object?